docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -25,6 +25,7 @@ metrics based on the tracing data.
   * [Jersey REST framework](https://github.com/eclipse-ee4j/jersey) 2.x -> 3.x
   * [Grizzly](https://github.com/eclipse-ee4j/grizzly) 2.3.x -> 4.x
   * [WebSphere Liberty](https://github.com/OpenLiberty/open-liberty) 23.x
+  * [Netty-http](https://github.com/netty/netty) 4.1.x
```

Comment:
```suggestion
  * [Netty HTTP](https://github.com/netty/netty) 4.1.x
```

And only 4.1, no 4.0.x?

Reply:
> And only 4.1, no 4.0.x?

When I was coding, I only considered the 4.1.x version. Just now I  included support for the 4.0.x version as well. There is something wrong with it, and I will check it.